### DISCUSSION TITLE
Code editor drag 'n drop functionality

### DISCUSSION THREAD

**Discussion by cristian-5** [zed-industries/zed#13862] [Feature Requests]
In VS-Code there is a drag 'n drop functionality over the selected text (in the editor), I would like to enable it on zed too, how do I do that?

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/support
